docs/features.md

```
@@ -147,4 +147,19 @@ tests:
 
 These tests can then be run with `cli test`. 
 
-Explore the [Schema](schema.md) for concrete definition of CLI configuration and manifest fields.
\ No newline at end of file
+Explore the [Schema](schema.md) for concrete definition of CLI configuration and manifest fields.
+
+
+## IDE Integration
+
+JSON schema validation provides real-time feedback, autocomplete suggestions, and type checking in your IDE. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)